Components/FlightControl/FlightTask.cpp

```diff
@@ -41,10 +41,10 @@ void FlightTask::InitTask()
  */
 void FlightTask::Run(void * pvParams)
 {
```

Comment:
Not sure if you do this already, but try to call TimerTransitions::Inst().Anything() -- maybe an Init()function in the constructor for FlightTask

Just to avoid the `new` call being implicitly called randomly down the line

Reply:
I remember already doing that but i cant seem to find it right now. Not sure if its called by Init Task. Ill have to check this

Reply:
Test and then edit this. this is a comment for myself